packages/react-native-otlp/test-project/android/settings.gradle

```diff
@@ -0,0 +1,45 @@
+import org.gradle.api.initialization.resolve.RepositoriesMode
+
+pluginManagement {
+    repositories {
+        google {
+            content {
+                includeGroupByRegex("com\\.android.*")
+                includeGroupByRegex("com\\.google.*")
+                includeGroupByRegex("androidx.*")
+            }
+        }
+        gradlePluginPortal()
+    }
+
+    // including external React Native plugin (from root of monorepo)
```

Comment:
based on the path this is just taking it from `test-project/node_modules` and not the root of the monorepo right?